clang/lib/Headers/lifetimebound.h

```diff
@@ -0,0 +1,19 @@
+/*===---- lifetimebound.h - Lifetime attributes -----------------------------===
+ *
+ * Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+ * See https://llvm.org/LICENSE.txt for license information.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ *===------------------------------------------------------------------------===
+ */
+
+#ifndef __LIFETIMEBOUND_H
+#define __LIFETIMEBOUND_H
+
+#define __lifetimebound [[clang::lifetimebound]]
```

Comment:
We should use GNU style attribute, e.g., `#define __lifetimebound __attribute__((lifetimebound))` so that it works for older C versions.

Reply:
Sounds good, fixed